code/datums/martial/wrestling.dm

```diff
@@ -1,6 +1,6 @@
 /*
 The contents of this file were originally licensed under CC-BY-NC-SA 3.0 as part of Goonstation(https://ss13.co).
-However, /tg/station and derivative codebases have been granted the right to use this code under the terms of the AGPL.
```

Comment:
What I said for bug_report.md's 9th and feedback_conversion_2017-11-12.py's 493rd line applies for this line.